api/managementpb/external.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+package management;
+
+option go_package = "managementpb";
+
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+import "google/api/annotations.proto";
+import "inventorypb/agents.proto";
+import "inventorypb/services.proto";
+import "protoc-gen-swagger/options/annotations.proto";
+
+// Add External
+
+message AddExternalRequest {
+  // Node identifier on which a external exporter is been running.
+  string runs_on_node_id = 1;
+  // Unique across all Services user-defined name. Required.
+  string service_name = 2 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // HTTP basic auth username for collecting metrics.
+  string username = 3;
+  // HTTP basic auth password for collecting metrics.
+  string password = 4;
+  // Scheme to generate URI to exporter metrics endpoints.
+  string scheme = 5;
+  // Path under which metrics are exposed, used to generate URI.
+  string metric_path = 6;
+  // Listen port for scraping metrics.
+  uint32 listen_port = 7 [
+    (validator.field) = {
+      int_gt: 0
+      int_lt: 65536
+    }
+  ];
+  // Node identifier on which a external service is been running.
+  string node_id = 8;
+  // Environment name.
+  string environment = 9;
+  // Cluster name.
+  string cluster = 10;
+  // Replication set name.
+  string replication_set = 11;
+  // Custom user-assigned labels for Service.
+  map<string, string> custom_labels = 15;
```

Comment:
Can it be 12?

Reply:
yes, but I'm pretty sure that we will add extra fields later.